net/extras/sample_aci_cfg.yml

```diff
@@ -33,9 +33,23 @@
 #APIC_EPG_BRIDGE_DOMAIN: "not_specified"
 #APIC_CONTRACTS_UNRESTRICTED_MODE: "no"
 
-## Full path of the leaf nodes connected to your servers.
-## List each node on a new line as shown commented
+# Topology information can be given following 3 ways.
```

Comment:
Please replace with the below.

Provide server to ACI physical connectivity information below. This information is used for establishing EPG bindings. You can specify node level binding, path level binding, or a mix of both as shown in examples.